BeerOverflowWindowsApp/GetBarList.cs

```diff
@@ -78,5 +78,15 @@ private void RateBarsButton_Click(object sender, EventArgs e)
             var ratingsForm = new BarRatingsForm();
             ratingsForm.Show();
         }
+
```

Comment:
Why are these classes left empty?

Reply:
Whoops, Designer did it automatically, I will remove it

Reply:
And those are methods, not classes. ^^

Reply:
You're right, sorry for this